### DISCUSSION TITLE
Time Duration of Verification for Student Verification Pack?

### DISCUSSION THREAD

**Discussion by sumukshashidhar** [community/community#22701] [New to GitHub]
<p>Hey, I’m currently a High School Junior and have applied for the GitHub Student Developer Pack. However, after 10 days of waiting for verification, I have received no email or status update as to what the status of my request is…</p>
<p>What is the typical waiting time for such a request for my scenario (my school does not have a .edu mail, and I’ve used my personal mail id)</p>
<p>If needed, I can submit additional proof, for perhaps a faster verification.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>HI <a class="mention" href="/sumukshashidhar">@sumukshashidhar</a>,</p>
<p>Thank you for being here and for your interest in the student developer pack, the current verification system does take a few weeks. I apologize for the delay and thank you for your patience.</p>

**Comment by mahendrark** [comment]
<p>The verification happened to me instantly, probably because my university’s email domain is easily recognized.</p>
<p>Hope yours will go through too, real fast. </p>

**Comment by ADJr01** [comment]
<p>hy,</p>
<p>I have applied for <strong>Github student pack</strong> almost one month ago.<em>Still no Response and no mail</em>.</p>
<p>Status showing :  <strong>Pending</strong></p>
<p>Any Solution?</p>

**Comment by clxsh** [comment]
<p>Hello,<br>
I applied for the Github student developer pack on Jul 20, 2020, its been 29 days and I haven’t got any response yet. It’s still in pending status.<br>
Could you help me on this</p>

**Comment by SohaibKhaliq** [comment]
hy i have applied for github student developer pack and i got verified, how long will it take to get my benfits.
![Web capture_5-7-2023_101734_education github com](https://github.com/community/community/assets/85817871/b87f999a-044b-421d-8c60-c3d306ddc643)

**Reply by jalandhar04** [reply]
Any update? Same problem with me! It displays this message "Congrats! Your request for [@jalandhar04](https://github.com/jalandhar04) was approved and your benefits will be available soon." It's been 2 days (verified as a student on Jul 7, 2023) for me, but i didn't get any benefits.

**Reply by balkrushnaborkar62** [reply]
Same here. Did you guys got access to benefits? It's been 3 days for me

**Reply by SohaibKhaliq** [reply]
Guys I have got the benefits 😌. But after 3-4 days
![Screenshot_20230711-161446_Edge](https://github.com/community/community/assets/85817871/b9009d1b-d87b-40ef-b441-6af952fe9138)

**Reply by sksaleha** [reply]
Bro any updates regading developer pack since 3 days gone no reply from  what happend to your developer pack?

**Comment by sharmavasant** [comment]
Hey, I'm currently in my college in pre final year 2 months back I applied for Students developer pack but got rejected due to some issues. Now I want to apply again but the website is not allowing may I know why?

**Comment by sharmavasant** [comment]
No vro it didn't came till now.

**Comment by kaizer1308** [comment]
its been 3 days for me huhu when i cna get my benefits
